src/modules/Shows.js

```diff
@@ -1,150 +1,186 @@
+    // handle updating the UI with every changes
+    static updateUI = async () => {
+      const shows = await Shows.getShows();
+      const card = document.getElementById('card');
+      const likes = await Shows.getLikesOrComments(this.likesURL);
+      const commentBody = document.getElementById('comment-section');
+      const tvShows = document.querySelector('#tv-shows p');
+      tvShows.innerHTML = `${'TV Shows ('}${shows.length})`;
+      card.innerHTML = '';
+      shows.forEach((show, index) => {
+        card.innerHTML += `
+                          <div class='container flex-column'>
+                              <div class='card-image'><img src='${
+                                show.image.medium
+                              }'></div>
+                              <div class='title-like'>
+                                  <div class='show-title'><p>${
+                                    show.name
+                                  }</p></div>
+                                  <div class='likes'>
+                                      <div id='heart${
+                                        show.id
+                                      }' class='icon ptr'></div>
+                                      <div class='likes-count'><p>${
+                                        likes.filter(
+                                          (like) => like.item_id - 1 === index,
+                                        )[0].likes
+                                      } likes</p></div>
+                                  </div>
+                              </div>
+                              <div class='card-comments'>
+                                  <button type='button' class='ptr' id='comment-${index}'>Comments</button>
                               </div>
                           </div>
-                          <div class='card-comments'>
-                              <button type='button' class='ptr' id='comment-${index}'>Comments</button>
-                          </div>
-                      </div>
-                  `;
-    });
+                      `;
+      });
 
-    // dynamically create comments for each listed shows on home page
-    shows.forEach((show, index) => {
-      // select the comment button
-      const commentButton = document.getElementById(`comment-${index}`);
-      // when the each comment button is clicked display the popup
-      commentButton.addEventListener('click', () => {
-        //   track which movie is selected to match the correct comment
-        Shows.globalIndex = index + 1;
-        // display comments on comments section
-        async function popuplateComments() {
-          const commenting = await Shows.getLikesOrComments(
-            `https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/tV364kOhzeIf5RoUn6sV/comments?item_id=${Shows.globalIndex}`,
+      // dynamically create comments for each listed shows on home page
```

Comment:
Please consider changing this comment as you are not creating a comment but implementing adding likes. ♻️